data/successors/successor side missions.txt

```diff
@@ -0,0 +1,488 @@
+			`	"... sweetest child ... never want ... not so ... somber current ... endure ..."`
+			`	It is not clear whether or not this represents the entire lyric content of the performance or simply if the translation device is unable to parse the more tonal phrases. The submerged singer displays a variety of colors and textures on their skin, changing with each successive phrase; green and rough on a splittingly high note, pink and smooth-looking on a strangely growl-like low section. Several in the crowd emulate this in their own skin patterning; the group pulses in color and pattern in time with the melody of the music.`
+			choice
+				`	(Continue listening.)`
+				`	(Return to your ship.)`
+					goto unappreciative
+			`	The music eventually crescendos with a long, droning wail of instrumentation accompanied by a rapid, high-pitched vocal performance that sees the water in the pool sloshing from side to side as the trio within move rapturously, circling around each other faster and faster before suddenly stopping. The performance comes to a close with a gentler and more playful section that sees the performers blowing bubbles to the surface of the pool and producing short, semi-random notes on their bizarre instruments.`
+			`	The Successors around you produce a dull, low hum after the piece's conclusion.`
+			choice
+				`	(Applaud.)`
+					goto applaud
+				`	(Try to hum yourself.)`
+					goto hum
+				`	(Return to your ship.)`
+					decline
+
+			label applaud
+			`	The Successors around you seem a little surprised or perhaps confused by your display, but do not express any offense at it, as far as you can tell. The trio of performers eventually emerge from the pool, gently tucking their instruments into various equally oddly-shaped cases before chatting with some of the few remaining members of the crowd and then departing.`
+				decline
+
+			label hum
+			`	The Successors around you seem amused by your attempt to emulate the humming; some briefly display a bluish-green wave of color across their skin before quickly returning to their base shade. Nonetheless, no one here seems to take any significant offense at the gesture. The performers eventually emerge from the pool, gently tucking their instruments into equally oddly-shaped cases before chatting with some of the few remaining members of the crowd and then departing.`
+				decline
+
+			label unappreciative
+			`	You return to your ship, gently extricating yourself from the throng of spellbound Successors, who continue to shimmer and sway in the distance until the performance eventually ends and they disperse to go their separate ways.`
+				decline
+
+mission `Successors: Stargazing 1`
+	minor
+	name "Transport Successor stargazers"
+	description "Transport a pair of Successor stargazers to <stopovers> and return them home to <destination> by <date>."
```

Comment:
What do you think? [Here](https://github.com/Daeridanii1/successors/pull/14#pullrequestreview-2142340636) you were saying to do `<planet stopovers>`, but that maybe shouldn't be the case here?
```suggestion
	description "Transport a pair of Successor stargazers to <planet stopovers> and return them home to <destination> by <date>."
```